README.md

```diff
@@ -285,16 +300,17 @@ from
         select
             x.wal,
             x.is_rls_enabled,
-            x.users,
+            x.subscription_ids,
             x.errors
         from
             realtime.apply_rls(
                 wal := w2j.data::jsonb,
                 max_record_bytes := 1048576
-            ) x(wal, is_rls_enabled, users, errors)
+            ) x(wal, is_rls_enabled, subscription_ids, errors)
     ) xyz
 where
     coalesce(pub.w2j_add_tables, '') <> ''
+    and array_length(xyz.subscription_ids, 1) > 0
```

Comment:
@olirice what's the performance impact of `array_length` as the size of `subscription_ids` increases? Since we're checking to see if there's at least one `subscription_id` can't we just access `subscription_ids` array at the first index and check if it's not null?

Reply:
no harm in swapping it out. replaced in 364d1a9